vite.config.js

```diff
@@ -0,0 +1,65 @@
+import { defineConfig } from 'vite'
+import vue from '@vitejs/plugin-vue2'
+import EnvironmentPlugin from 'vite-plugin-environment'
+import { NgmiPolyfill } from "vite-plugin-ngmi-polyfill"
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const path = require('path')
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const fs = require('fs')
+const packageJson = fs.readFileSync('./package.json')
+const appName = JSON.parse(packageJson).appName
+const appVersion = JSON.parse(packageJson).version
+const sbcName = JSON.parse(packageJson).sbcName
+const sbcVersion = JSON.parse(packageJson).dependencies['sbc-common-components']
+const aboutText1 = (appName && appVersion) ? `${appName} v${appVersion}` : ''
+const aboutText2 = (sbcName && sbcVersion) ? `${sbcName} v${sbcVersion}` : ''
+
+// https://vitejs.dev/config/
+export default defineConfig(() => {
+  return {
+    define: {
+      'import.meta.env.ABOUT_TEXT':
+        (aboutText1 && aboutText2) ? `"${aboutText1}<br>${aboutText2}"`
+          : aboutText1 ? `"${aboutText1}"`
+            : aboutText2 ? `"${aboutText2}"`
+              : ''
+    },
+    envPrefix: 'VUE_APP_', // Need to remove this after fixing vaults. Use import.meta.env with VUE_APP.
+    plugins: [
+      vue(),
+      EnvironmentPlugin({
+        BUILD: 'web' // Fix for Vuelidate, allows process.env with Vite.
+      }),
+      // Fix Module has been externalized for browser compatibility warning.
+      NgmiPolyfill()
```

Comment:
There was a warning in the console ONLY when running locally:
![only in local](https://github.com/bcgov/business-edit-ui/assets/122301442/38363549-3542-42fe-b720-e271af0039f6)

This fixes all of them except for source-map-js warning. When running locally now:
![source-map-js](https://github.com/bcgov/business-edit-ui/assets/122301442/1d82e7d1-8099-45f4-9fd3-5a26394e0229)